assets/js/components/settings/SettingsActiveModule/ConfirmDisconnect.js

```diff
@@ -71,9 +71,9 @@ export default function ConfirmDisconnect( { slug, handleDialog } ) {
 
 		if ( ! error ) {
 			clearWebStorage();
-			global.location.assign( adminReauthURL );
+			global.location.assign( dashboardURL );
```

Comment:
This should use the `core/location` data store instead.

Then, related to this, we also need to make sure the dialog remains open until the navigation/redirect is complete (basically it just shouldn't close). See https://github.com/google/site-kit-wp/issues/2183#issuecomment-823985934